spec/units/blockchain/consensus.cr

```diff
@@ -28,11 +29,11 @@ describe Consensus do
 
   describe "difficulty" do
     it "should return the #difficulty_at for the blockchain" do
-      difficulty_at(0_i64).should eq(4)
+      difficulty_at(0_i64).should eq(3)
```

Comment:
Oops, that's my bad! 😢 
I set a easy difficulty for the debugging. It should be same as before :pray:
I'll revert here!

Reply:
cool